privaterelay/templates/includes/premium-onboarding.html

```diff
@@ -0,0 +1,173 @@
+            <div class="mzp-l-content mzp-t-content-xl">
+                <div class="c-premium-onboarding-headlines">
+                    <h1>Block, Forward, and Reply!</h1>
+                    <h2>Now you can control what hits your inbox, one email at a time.</h2>
+                </div>
+
+                <div class="c-premium-onboarding-content">
+                    <div class="c-premium-onboarding-content-image">
+                        <img src="{% static 'images/dashboard-onboarding/woman-email.svg' %}" alt="Woman next to large envelope">
+                    </div>
+                    <div class="c-premium-onboarding-content-description">
+                        <p><strong>Reply to emails without giving away your real address:</strong> Need to reply to emails sent to an alias? Just reply as normal—your alias will still protect your email address.</p>
+                        <p><strong>Get the Relay extension for Firefox</strong> The Relay extension for Firefox makes using email aliases even easier. Reply to emails without giving away your real address: Need to reply to emails sent to an alias? Just reply as normal—your alias will still protect your email address.</p>
+                        <a target="_blank" href="https://addons.mozilla.org/firefox/addon/private-relay/?utm_source=fx-relay&utm_medium=onboarding&utm_campaign=install-addon" class="c-premium-onboarding-install-addon mzp-c-button mzp-t-product">Get Relay extension</a>
+                        <div class="c-premium-onboarding-action-completed">
+                            <img class="margin-right" src="/static/images/icon-check.svg" alt="">
+                            Relay extension installed!
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </li>
+    </ol>
+    <div class="c-premium-onboarding-progress">
+        <div class="mzp-l-content mzp-t-content-xl">
+            <div class="c-premium-onboarding-actions-wrapper">
+                <!-- Step 1 -->
+                <div class="c-premium-onboarding-actions c-premium-onboarding-action-step-1  {% if user_profile.onboarding_state == 0 %} is-visible {% endif %}">
+                    <button class="mzp-c-button mzp-t-product js-premium-onboarding-next-step"
+                    data-ga="send-ga-pings"
+                    data-event-category="Premium Onboarding"
+                    data-event-label="onboarding-step-1-continue"
+                    data-event-value="1"
+                    data-event-nonInteraction="false"
+                    >Get Started</button>
+                </div>
+                <!-- Step 2 -->
+                <div class="c-premium-onboarding-actions c-premium-onboarding-action-step-2 {% if user_profile.onboarding_state == 1 %} is-visible {% endif %}">
+                    {% if not user_profile.subdomain %}
```

Comment:
In Steps 2 and 3, there are two buttons (only one shows at a time): skip and continue once the action is complete

If a user already completed the action on page load, we do not even render the skip button. We still render the continue button regardless of state because if they complete the action, we'll want it.